src/scenes/home/landing/hero/hero.css

```diff
@@ -0,0 +1,77 @@
+.heroContainer {
+  display: flex;
+  flex-flow: column;
+}
+
+.pageHeading {
+  background-color: #191f39;
+  color: #ffffff;
+  text-align: center;
```

Comment:
The `background-color` and `color` rules can be removed here, let's just keep the standard white background-color and the standard color scheme for now.